### PR TITLE
Fix: Circular dependency

### LINKED ISSUE #50: Circular dependency
### Subject of the issue
Getting a CLI warning when running rub.

**Circular dependency: src/extensions/adapt-scrollSnap/js/Models.js -> src/extensions/adapt-scrollSnap/js/Views.js -> src/extensions/adapt-scrollSnap/js/Models.js**

Not sure if this is a rub issue or a Scrollsnap issue.

### Your environment
* Framework 5.28.6
* ScrollSnap 5.1.4

### Steps to reproduce
1. Enable Scrollsnap
2. Rub `rub -ds`

### PR DESCRIPTION
**Pull request by oliverfoster**
fixes #50 

### Fix
* Removed reference to Views from Models

### CODE REVIEW

File: js/Models.js [hunk 1]
```diff
@@ -94,7 +93,10 @@ export default class Models {
   static shouldStopRendering(model) {
     const articlesAndBlocks = this.articlesAndBlocks;
     let stepLockIndex;
-    if (!Views.isScrollSnapActive && State.isTrickleEnabled) {
+    const isScrollSnapActive = Config.canUseScrollSnap &&
```
Comment (swashbuck) [inline js/Models.js:96]: Trailing space

Comment (oliverfoster) [reply js/Models.js:96]: Feel free to edit and merge.

**Review by swashbuck — APPROVED** [review]
👍

**Comment by oliverfoster** [comment]
That should fix it.

**Pull request merged**